rootfs/etc/nginx/lua/monitor.lua

```diff
@@ -46,6 +46,8 @@ local function metrics()
     upstreamResponseLength = tonumber(ngx.var.upstream_response_length) or -1,
     --upstreamStatus = ngx.var.upstream_status or "-",
   }
+  local payload, err = cjson.encode(m)
```

Comment:
Should check the `err`.

Reply:
@tokers  If there is an error, how to deal with this error？ only recode log?

Reply:
done.